backend/src/routes.ts

```diff
@@ -225,6 +225,7 @@ if (config.experimental.v2) {
   server.post('/api/v2/schemas', ...postSchemaV2)
 
   server.get('/api/v2/reviews', ...getReviews)
+  server.get('/api/v2/review/:reviewId', ...getReviews)
```

Comment:
I think this endpoint was added by mistake for the demo but isn't actually needed?